layouts/results-page/single.html

```diff
@@ -25,7 +25,74 @@ <h2>Warning: This page requires Javascript.</h2>
     {{ partial "ad-autorefresh.html" . }}
 
     <button type="button" class="js-key-contests-btn btn btn-primary btn-lg">Refresh</button>
+    <div
```

Comment:
If we move the find races to the top, will people think to scroll down to the other races? We might still want a nav or something to let them know the other races are there.

Reply:
A nav is next on my list. The find races is small, so most screens will be able to see key races (even on mobile).

Reply:
👍